lib/content-services/src/lib/content-node-share/content-node-share.dialog.spec.ts

```diff
@@ -176,7 +176,7 @@ describe('ShareDialogComponent', () => {
 
         fixture.detectChanges();
 
-        fixture.nativeElement.querySelector('.mat-slide-toggle label')
+        fixture.nativeElement.querySelector('[data-automation-id="adf-share-toggle"] label')
```

Comment:
This: 
fixture.nativeElement.querySelector('[data-automation-id="adf-share-toggle"] label')
            .dispatchEvent(new MouseEvent('click'));

is also repeated couple times, please move it to function and reuse too